### ISSUE TITLE
Sveltekit -  TypeError: canvas.createCanvas is not a function

### ISSUE THREAD

**Issue by magick93** [pikasojs/pikaso#57]
I'm trying to use Pikaso with Sveltekit. I've tried several variations of the following but all generally result in the same error.

Any direction is greatly appreciated. 

```
<script lang="ts">
    import Pikaso from 'pikaso'
    import { onMount } from 'svelte';
    let editor;

    onMount(async () => {

        let pikaso = new Pikaso({
        container: editor,
      });

      pikaso.shapes.polygon.insert({
        x: 250,
        y: 250,
        radius: 900,
        sides: 6,
        fill: "tomato",
      });
    });


    
</script>

<div bind:this={editor}>
   
</div>
```

## Error

```
Uncaught (in promise) TypeError: canvas.createCanvas is not a function
    createCanvasElement index.all.js:10383
    Canvas$1 index.all.js:1614
    SceneCanvas index.all.js:1676
    _buildDOM index.all.js:4161
    Stage index.all.js:3615
    Board2 index.all.js:12137
    init index.all.js:14282
    Pikaso2 index.all.js:14219
    instance Canvas.svelte:8
    run index.mjs:18
    mount_component index.mjs:1802
    flush index.mjs:1067
    init index.mjs:1894
    Root root.svelte:920
    createProxiedComponent svelte-hooks.js:341
    ProxyComponent proxy.js:242
    Proxy<Root> proxy.js:349
    initialize start.js:702
    _hydrate start.js:1602
    start start.js:1639
    <anonymous> (index):230
```

**Comment by raminious** [comment]
@magick93 I have no experience with Svelte. However, I'll take a look and try to figure out what's happening there

**Comment by raminious** [comment]
There is something wrong with the Rollup build. I could fix that, but I would need more time to investigate the problem and make sure the fix is correct. I will release the patch with v2.4.1 in one or two days.

**Comment by magick93** [comment]
Thanks @raminious

**Comment by raminious** [comment]
@magick93 The bug has been fixed and published with version 2.4.1   
Also, regarding to issue #56, you don't need to set `global` anymore.

[https://github.com/pikasojs/pikaso/releases/tag/v2.4.1](https://github.com/pikasojs/pikaso/releases/tag/v2.4.1)

**Comment by magick93** [comment]
Thanks @raminious - that is really awesome!
